operator/internal/manifests/openshift/var.go

```diff
@@ -37,6 +37,14 @@ func clusterRoleName(opts Options) string {
 	return opts.BuildOpts.GatewayName
 }
 
+func monitoringRoleName(stackName string) string {
+	return fmt.Sprintf("%s-metrics-reader", stackName)
+}
+
+func monitoringRoleBindingName(stackName string) string {
+	return fmt.Sprintf("read-%s-metrics", stackName)
```

Comment:
I think this made sense when the binding was a non-namespaced object, but I think now the usual naming convention can be used.